ee/clickhouse/views/events.py

```diff
@@ -100,7 +101,12 @@ def list(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         team = self.team
         filter = Filter(request=request, team=self.team)
 
-        query_result = self._query_events_list(filter, team, request, limit=limit)
+        try:
+            query_result = self._query_events_list(filter, team, request, limit=limit)
+        except CHQueryErrorCannotCompileRegexp:
+            raise ValidationError(
+                "Invalid filter for events. Check that your selectors and regexes are defined properly."
```

Comment:
This is misleading - we fail to compile legit selectors on our end. This is not an user error.